test/assets/CalcBreakdownTestConstants.scala

```diff
@@ -28,6 +28,24 @@ object CalcBreakdownTestConstants {
     crystallised = true
   )
 
+  val calculationBillTaxableIncomeZeroModel = Calculation(
+    totalTaxableIncome = Some(0.00),
+    crystallised = true,
+    nationalRegime = Some("UK")
+  )
+
+  val calculationBillWelshModel = Calculation(
+    totalTaxableIncome = Some(0.00),
+    crystallised = true,
+    nationalRegime = Some("Welsh")
```

Comment:
National regime is not "Welsh"
Possible values are "UK", "Scotland", "Wales"

Reply:
This could potentially have been identified earlier with a small change to messages, discuss with me